### PR TITLE
e2e/kilo-kind-userspace.yaml: fix boringtun cli flags

### PR DESCRIPTION
**Pull request by leonnicolas**
A change in boringtun's cli caused the boringtun container to crash.

Signed-off-by: leonnicolas <leonloechner@gmx.de>

### CODE REVIEW

File: manifests/kilo-kubeadm-usespace-heterogeneous.yaml [hunk 1]
```diff
@@ -0,0 +1,365 @@
+apiVersion: v1
```
Comment (squat) [inline manifests/kilo-kubeadm-usespace-heterogeneous.yaml:1]: Is this used in the e2e tests or is it just for our benefit?

**Review by squat — COMMENTED** [review]
This looks really good and indeed the tests make it further now, but there's still a failure:

https://github.com/squat/kilo/runs/7223221087?check_suite_focus=true#step:4:353

File: manifests/kilo-kubeadm-usespace-heterogeneous.yaml (hunk 1 above)
Comment (leonnicolas) [reply manifests/kilo-kubeadm-usespace-heterogeneous.yaml]: not sure if that one works. I added it on accident. I removed it again.

**Review by squat — DISMISSED** [review]
Very very nice catches Leon! 🎉🌮

**Review by squat — APPROVED** [review]

**Pull request merged**